scalafmt-tests/src/test/resources/default/Try.stat

```diff
@@ -58,9 +58,71 @@ object Foo {
     bar()
   }
 }
+<<< try/catch/finally curly no single line blocks
+object Foo {
+  try { foo()  }
+  catch { case bar => baz }
+  finally {  qux()  }
+}
+>>>
+object Foo {
+  try {
+    foo()
+  } catch { case bar => baz }
```

Comment:
This formatting confusing me a little. Current behavior is not better:
```scala
object Foo {
  try {
    foo()
  } catch { case bar => baz } finally {
    qux()
  }
}
```

Should scalafmt format it like this?

```scala
// option 1
object Foo {
  try { foo() }
  catch { case bar => baz }
  finally { qux() }
}
// option 2
object Foo {
  try {
    foo()
  } catch {
    case bar => baz
  } finally { 
    qux()
  }
}
```

Anyway this is not a regression because current formatting is more stranger

Reply:
@poslegm i have already implemented and will submit option 1, but was planning to do it separately, after this change is merged (as it claims to treat try/finally as if/else).

let me know if you want me to include it here (as it also modifies if/else).

Reply:
This PR does not introduce a regression for this particular case so I'm fine with merging as is and we can followup separately to use option 1 or 2 (I don't have a preference)

Reply:
It's great! Better to do it separately